Sources/ObjCSupport/LOKInsetLayout.swift

```diff
@@ -9,12 +9,23 @@
 import CoreGraphics
 
 @objc open class LOKInsetLayout: LOKBaseLayout {
+    @objc public let insets: EdgeInsets
+    @objc public let alignment: LOKAlignment?
+    @objc public let viewClass: View.Type?
+    @objc public let sublayout: LOKLayout
+    @objc public let configure: ((View) -> Void)?
+
     @objc public init(insets: EdgeInsets,
                       alignment: LOKAlignment? = nil,
                       viewReuseId: String? = nil,
                       viewClass: View.Type? = nil,
                       sublayout: LOKLayout,
                       configure: ((View) -> Void)? = nil) {
+        self.insets = insets
+        self.sublayout = sublayout
+        self.alignment = alignment
+        self.viewClass = viewClass
```

Comment:
Since this implementation provides defaults for some parameters `alignment` and `viewClass`, it would be good to have the properties reflect that.

Since that would turn some optional properties into non-optional, that'd be a breaking change so it'd be better to get it done with this change, so we don't have to make a breaking change later.

Reply:
Thanks for pointing this out. Done the required changes.